### PR TITLE
Inform user about all the implication related to unlinking citations

### PR DESCRIPTION
**Pull request by pedropintosilva**
Add also notification once they are unlinked

Signed-off-by: Pedro Pinto Silva <pedro.silva@collabora.com>
Change-Id: Id26f7228f5a845b79f941256738a0d83bd12e43f

### CODE REVIEW

**Comment by pedropintosilva** [comment]
unlink-citations-warn
unlink-citations-unlinked

**Comment by eszkadev** [comment]
Added "confirm" handler with the placeholder TODO where feature should be implemented

File: browser/src/control/Control.Zotero.js [hunk 1]
```diff
@@ -1036,6 +1036,32 @@ L.Control.Zotero = L.Control.extend({
...
+					'value': 'vnd.oasis.opendocument.field.UNHANDLED'
+				},
+				'FieldCommandPrefix': {
+					'type': 'string',
+					'value': 'ADDIN ZOTERO_'
+				},
+			};
+			command = '.uno:DeleteTextFormFields';
+		}
+		this.map.sendUnoCommand(command, parametes);
```
Comment (eszkadev) [inline browser/src/control/Control.Zotero.js:1055]: @lpranam what if it is a different type?

Comment (lpranam) [reply browser/src/control/Control.Zotero.js:1055]: Core APIs are still in development for others

**Comment by lpranam** [comment]
unlinking of section (bibliography with refmarks) remains. core side work is in progress but regardless this can merge now.

**Review by eszkadev — APPROVED** [review]

**Pull request merged**